src/cpp/src/visual_language/qwen2vl/classes.cpp

```diff
@@ -290,9 +299,129 @@ ov::Tensor merge_text_and_image_embeddings(
+    //    grid_h / spatial_merge_size, spatial_merge_size, patch_size,
+    //    grid_w / spatial_merge_size, spatial_merge_size, patch_size]
+    //
+    //  transpose 
+    //
+    // [grid_t, grid_h / spatial_merge_size, grid_w / spatial_merge_size, spatial_merge_size, spatial_merge_size,
+    //  temporal_patch_size * channel, patch_size, patch_size ]
+    //
+    //  reshape
+    //
+    // [grid_t * (grid_h / spatial_merge_size) * (grid_w / spatial_merge_size) * (spatial_merge_size * spatial_merge_size),
+    //
+    //  temporal_patch_size, channel,
+    //
+    //  (patch_size * patch_size)]
+    //
+    //
+    //
+    // [grid_t * grid_h * grid_w, channel * config.temporal_patch_size * config.patch_size * config.patch_size]
+
+    auto raw_images = std::make_shared<ov::op::v0::Parameter>(ov::element::u8, ov::PartialShape{-1, -1, -1, -1});
+    auto target_shape = std::make_shared<ov::op::v0::Parameter>(ov::element::i64, ov::PartialShape{2});
+    auto image_mean = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, ov::PartialShape{1, -1, 1, 1});
+    auto image_scale = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, ov::PartialShape{1, -1, 1, 1});
+
+    auto broadcast_shape = std::make_shared<ov::op::v0::Parameter>(ov::element::i64, ov::PartialShape{4});
+    auto temp_shape8d = std::make_shared<ov::op::v0::Parameter>(ov::element::i64, ov::PartialShape{8});
+    auto temp_shape4d = std::make_shared<ov::op::v0::Parameter>(ov::element::i64, ov::PartialShape{4});
+    auto last_shape = std::make_shared<ov::op::v0::Parameter>(ov::element::i64, ov::PartialShape{2});
+
+    raw_images->set_friendly_name("raw_images");
+    raw_images->output(0).get_tensor().set_names({"raw_images"});
+    
+    target_shape->set_friendly_name("target_shape");
+    target_shape->output(0).get_tensor().set_names({"target_shape"});
+
+    image_mean->set_friendly_name("image_mean");
+    image_mean->output(0).get_tensor().set_names({"image_mean"});
+
+    image_scale->set_friendly_name("image_scale");
+    image_scale->output(0).get_tensor().set_names({"image_scale"});
+
+    broadcast_shape->set_friendly_name("broadcast_shape");
+    broadcast_shape->output(0).get_tensor().set_names({"broadcast_shape"});
+
+    temp_shape8d->set_friendly_name("temp_shape8d");
+    temp_shape8d->output(0).get_tensor().set_names({"temp_shape8d"});
+    temp_shape4d->set_friendly_name("temp_shape4d");
+    temp_shape4d->output(0).get_tensor().set_names({"temp_shape4d"});
+    last_shape->set_friendly_name("last_shape");
+    last_shape->output(0).get_tensor().set_names({"last_shape"});
+
+    // res.buf[i] = ((float(v2) / 255.0f) - m3[c]) / s3[c];
+    auto raw_images_planar = std::make_shared<ov::op::v1::Transpose>(raw_images, 
+        std::make_shared<ov::op::v0::Constant>(element::i32, Shape{4}, std::vector<int32_t>{0, 3, 1, 2})
+    );
+    ov::op::v0::Interpolate::Attributes attrs = {
+        .axes = {2,3},
+        .mode = "cubic"
+    };
```

Comment:
```suggestion
    ov::op::v0::Interpolate::Attributes attrs = { };
    attrs.axes = {2,3};
    attrs.mode = "cubic";
```
std:c++17 doesn't support designated initializers
classes.cpp(364): error C7555: use of designated initializers requires at least '/std:c++20'